src/base/bittorrent/sessionimpl.cpp

```diff
@@ -1258,7 +1260,8 @@ void SessionImpl::setGlobalMaxSeedingMinutes(int minutes)
 
     if (minutes != globalMaxSeedingMinutes())
     {
-        m_globalMaxSeedingMinutes = minutes;
+        m_globalMaxSeedingMinutes =
+                std::clamp(minutes, minutes, BitTorrent::Torrent::MAX_SEEDING_TIME);
```

Comment:
So why not apply lower limit as well?
```suggestion
                std::clamp(minutes, -1, BitTorrent::Torrent::MAX_SEEDING_TIME);
```

Reply:
Applied  BitTorrent::Torrent::NO_SEEDING_TIME_LIMIT and BitTorrent::Torrent::NO_INACTIVE_SEEDING_TIME_LIMIT constants